cgmanifest.json

```diff
@@ -25103,8 +25103,8 @@
         "type": "other",
         "other": {
           "name": "python-xmltodict",
-          "version": "0.12.0",
-          "downloadUrl": "https://github.com/martinblech/xmltodict/archive/v0.12.0/xmltodict-0.12.0.tar.gz"
+          "version": "0.13.0",
+          "downloadUrl": "https://github.com/martinblech/xmltodict/archive/refs/tags/v0.13.0.tar.gz"
```

Comment:
There is a PR check failing because of the change of downloadUrl. Please use the URL https://github.com/martinblech/xmltodict/archive/v0.13.0/xmltodict-0.13.0.tar.gz

Reply:
Done.